sphinx/config.py

```diff
@@ -631,10 +631,10 @@ def _substitute_copyright_year(copyright_line: str, replace_year: str) -> str:
     Legal formats are:
 
     * ``YYYY``
-    * ``YYYY,``
-    * ``YYYY ``
-    * ``YYYY-YYYY,``
-    * ``YYYY-YYYY ``
+    * ``YYYY-YYYY``
+    * ``YYYY–YYYY``
```

Comment:
Even though it would be possible to disable `RUF002` (ambiguous character) linting on a per-file basis, I think it's a useful and important enough lint that I'll rewrite this docstring to refer to EN-DASH by character name instead.